hook.c

```diff
@@ -290,7 +313,7 @@ int run_hooks_opt(struct repository *r, const char *hook_name,
 	};
 
 	/* Interject hook behavior depending on strategy. */
-	if (r && r->gitdir &&
+	if (r &&
 	    handle_hook_replacement(r, hook_name, &options->args))
```

Comment:
Git code style question/comment:

Can the call to `handle_hook_replacement` be moved up on the same line as `if (r &&`?

Or is it git coding convention to make function calls in conditions on their own line?